lib/mix/lib/mix/tasks/escript.install.ex

```diff
@@ -121,13 +126,15 @@ defmodule Mix.Tasks.Escript.Install do
 
       # If existing executable didn't change but it is not the one we installed,
       # it is a conflict
-      previous_executable && previous_executable != dst ->
+      previous_executable && not equal_with_dot_bat previous_executable, dst  ->
```

Comment:
Can we add parentheses for clarity here and below?

    not equal_with_dot_bat(previous_executable, dst)